packages/astro/src/core/create-vite.ts

```diff
@@ -155,6 +156,7 @@ export async function createVite(
 			astroDevToolbar({ settings, logger }),
 			vitePluginFileURL({}),
 			astroInternationalization({ settings }),
+			astroEnvVirtualModPlugin({ settings, logger })
```

Comment:
Only a minor nit, but since this is a Vite pre plugin, it would be nice to put this higher up the list as I think its hook should get higher priority?

Reply:
i looked at the plugin above (i18n) and it's also a pre plugin so I assumed there was fine. Happy to move it higher but how much?